nml/actions/action0properties.py

```diff
@@ -353,8 +367,40 @@ def ctt_list(prop_num, *values):
     ]
 
 
-def VariableWordListProp(num_prop, data, extended=False):
-    return VariableListProp(num_prop, data, 2, extended)
+def VariableWordListProp(num_prop, data, size_size=1):
+    return VariableListProp(num_prop, data, 2, size_size)
+
+
+def badge_list(prop_num, *values):
+    # values may have multiple entries, if more than one item ID is set (e.g. multitile houses)
+    # Each value is an expression.Array of cargo types
```

Comment:
Might this `Array of cargo types` be a remnant of copying something? Or what have cargo types to do with badges?